lib/Textile/API/index.ts

```diff
@@ -89,8 +73,9 @@ class API {
     return result as string
   }
 
-  addFiles = async (dir: IDirectory, threadId: string, caption?: string): Promise<BlockInfo> => {
-    const byteArray = Directory.encode(dir).finish()
+  addFiles = async (dir: pb.Directory.AsObject, threadId: string, caption?: string): Promise<BlockInfo> => {
+    const directoryInstance = convertDirectoryData(dir)
+    const byteArray = directoryInstance.serializeBinary()
```

Comment:
In this files, you can see what using the new pb types looks like. Pretty simple, not a big change.